broker/lib/fabrik/DirectorInstance.js

```diff
@@ -213,7 +225,66 @@ class DirectorInstance extends BaseInstance {
       });
   }
 
+  update20(params) {
```

Comment:
whats diff between update20 and update10

Reply:
https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/304/files#diff-46e37f17222369258c2e785ea9671484R188